tests/integration/security/testdata/authz/v1beta1-conditions.yaml.tmpl

```diff
@@ -140,6 +214,30 @@ spec:
       values: ["1.2.3.4"]
 ---
 
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: condition-destination-ip-notvalues
+  namespace: "{{ .NamespaceC }}"
+spec:
+  selector:
+    matchLabels:
+      app: c
+  rules:
+  - to:
+    - operation:
+        paths: ["/destination-ip-notValues-a"]
+    when:
+    - key: destination.ip
+      notValues: ["{{ .IpA }}", "{{ .IpB }}"]
+  - to:
```

Comment:
there are 2 `to` in the same policy, how do you know a request is denied by which one?

Reply:
the 2 `to` have different operation paths.